products/postmarketos.md

```diff
@@ -0,0 +1,75 @@
+---
+title: postmarketOS
+category: os
+tags: linux-distribution
+#iconSlug: postmarketOS
+permalink: /postmarketos
+versionCommand: cat /etc/os-release
+releasePolicyLink: https://wiki.postmarketos.org/wiki/Releases
+eolColumn: Security Support
+eolWarnThreshold: 7
+activeSupportWarnThreshold: 7
+releaseColumn: Latest
+releaseDateColumn: true
+
+auto:
+  methods:
+    # Cross check with the channels.cfg
+    # https://gitlab.com/postmarketOS/pmaports/-/blob/master/channels.cfg?ref_type=heads
```

Comment:
Are dates on https://distrowatch.com/table.php?distribution=postmarketos corrects (at least roughly) ? I ask because activating this configuration will make it impossible to set the `latestReleaseDate` manually (auto update will always overwrite what's being set manually).

Reply:
Roughly. The date next to `23.12 ̇ is `2023-12-19`, and the actual release date was `2023-12-18`. Should we turn this off perhaps?

Reply:
+- a few days should be okay.

Reply:
Then I would keep this as is.